src/swagger/http/views/swagger.ecr

```diff
@@ -4,7 +4,7 @@
   <head>
     <meta charset="UTF-8">
     <title><%= title %></title>
-    <link rel="stylesheet" type="text/css" href="/swagger-ui/swagger-ui.css" >
+    <link rel="stylesheet" type="text/css" href="//cdn.jsdelivr.net/npm/swagger-ui-dist@3/swagger-ui.css" >
     <link rel="icon" type="image/png" href="/swagger-ui/favicon-32x32.png" sizes="32x32" />
     <link rel="icon" type="image/png" href="/swagger-ui/favicon-16x16.png" sizes="16x16" />
```

Comment:
The icons are also available in the CDN.

Reply:
You got me.